server/bleep/src/webserver/answer.rs

```diff
@@ -490,9 +678,18 @@ fn grow(doc: &ContentDocument, snippet: &Snippet, size: usize) -> Option<String>
     Some(content[new_start_byte..new_end_byte].to_owned())
 }
 
+fn get_keywords(query: &str) -> String {
+    let stop_words_path = "../../../server/stopwords.txt";
```

Comment:
this path will not always resolve. i propose that we use `include_str!` here and bake stopwords into the binary (just 3.6Kb). 

and then calculate `sw` in the next line as:

```
let stop_words: HashSet<String> = sw.lines().map(ToOwned::to_owned).collect();
let sw = stop_words.into();
```